src/course-home/data/api.js

```diff
@@ -155,6 +155,7 @@ export function normalizeOutlineBlocks(courseId, blocks) {
           title: block.display_name,
           hideFromTOC: block.hide_from_toc,
           navigationDisabled: block.navigation_disabled,
+          isPreview: block.is_preview,
```

Comment:
Ideally, this should be reflected in tests but, since this entire function is missing tests currently, I'll let it pass.